Makefile

```diff
@@ -80,10 +87,8 @@ lint:
 ### Docker Images
 
 .PHONY: docker-component # Not intended to be used directly
-docker-component: check-component exe
-	docker build -t grafana/$(COMPONENT) -f ./cmd/$(COMPONENT)/Dockerfile .
-	docker tag grafana/$(COMPONENT) $(COMPONENT)
-	rm ./cmd/$(COMPONENT)/$(COMPONENT)
+docker-component: check-component
+	docker buildx build $(BUILDX_OUTPUT) --platform=$(PLATFORMS) -t $(NAMESPACE)/$(COMPONENT) -f ./cmd/$(COMPONENT)/Dockerfile .
```

Comment:
docker tag alias here have issue for when set `BUILDX_OUTPUT=--push`
means no local image for tag.